### DISCUSSION TITLE
How do you change fonts?

### DISCUSSION THREAD

**Discussion by jvvk** [quarto-dev/quarto-cli#797] [Q&A] [answered]
How can I change the default fonts?

**Comment by dragonstyle** [comment, marked as answer]
Hi,

Check here for a variety of font options that work across many formats. 

https://quarto.org/docs/reference/formats/html.html#fonts

Check with your target output reference page specifically to consider whether these options work or whether there are additional or alternative options that make sense.

HTH!

Charles

**Reply by jvvk** [reply]
Thank you!
